style2.css

```diff
@@ -1157,15 +1015,37 @@ textarea {
     width: 369px;
   }
 
+  .type-of-cards {
```

Comment:
- [ ] Please correct the overlapping as requested by the previous reviewer. Stick to the Figma template is a project requirement.
Your portfolio:
<img width="488" alt="Screen Shot 2022-08-27 at 7 53 34 PM" src="https://user-images.githubusercontent.com/47673311/187050898-2080fbab-bb4d-429e-a79a-ffd0368878fe.png">

Figma template:
<img width="706" alt="Screen Shot 2022-08-27 at 7 54 53 PM" src="https://user-images.githubusercontent.com/47673311/187050905-70622fb7-37f6-484d-9023-223b685d33ce.png">

Reply:
@ElsonOtake ok i have been trying all day i even rebuilt the whole site do you have any advise on how to do it

Reply:
and now it is responsive I dont want next week to only resize the page

Reply:
- [ ] Kindly address this 👍🏻